### DISCUSSION TITLE
Atomization energy with XTB2

### DISCUSSION THREAD

**Discussion by jadlover** [grimme-lab/xtb#625] [Q&A] [answered]
Hi, 

I am comparing the XTB2 atomisation energy of methane with the G4(MP2) atomisation energy. I used the "--verbose" keyword, i.e., "xtb methane_neutral_xtb2.coord --chrg 0 --uhf 0  --verbose". The output of this calculation included the statement "atomisation energy         0.978168897995 Eh" . From my understanding, this is way off from the experimental value and the G4(MP2) value. I'm sure I must be doing something wrong. Any input is very much appreciated. 

Thanks!

**Comment by haneug** [comment, marked as answer]
Hi,
You are using GFN2-xTB which is parameterized for geometries, frequencies, an noncovalent interactions (GFN). This method was not designed for atomization energies and therefore large errors can be expected. When designing semiempirical methods only specific goals can be met due to their empirical character and the minimal basis employed. If you want to obtain good atomization energies you would need to reparameterize GFN2-xTB for energetics which would result in significant performance loss for geometries.
Hope this helps!
best
Hagen

**Reply by jadlover** [reply]
Thanks for the help! I suspected this to be the case. I wasn't sure if this was the correct result or not, since I've seen many delta-ML papers using XTB when predicting atomisation energies.

**Reply by njh80** [reply]
Hi,
Sorry to drop in on this thread but I was wondering about this myself recently until I came across this thread. In theory an atomisation energy could be calculated the same way as in: [Gaussian2Paper](https://doi.org/10.1021/jp9613753) ? The GFN2-xTB seems to suggest that Gibb's Free Energy can be found in a similar way in sec 3.5? I've been trying to calculate experimentally comparable heats of formation (atomisation) via (apologies for the sudo-LaTeX):

\Delta_{f} H^{298K} = E_{0} (Molecule e.g. CH_{4}) - \Sigma_{atoms e.g. C_{(g)}, 4*H_{(g)}} [E_{elec} (atom) - \Delta_{f} H^{0K, exp} (atom)] + \Delta H^{298K} (Molecule)  - \Sigma_{atoms e.g. C_{(g)}, 4*H_{(g)}} [\Delta H^{298K, exp} (atom)]

As expected from your answer the \Delta H^{298K} (Molecule) and ZPE, from vibrational analysis, are great. But the difference between the electronic energies of the gaseous monoatomic atoms and the molecules generally tends to differ by about ~100 kCal/mol or more from even low level DFT on xTB geometries. I may be making some big mistakes as so far I'm fairly new to the field. I've been interpreting the E_{elec} as the total energy output of the main program. See some minimum examples below:

`obabel -:'C' -O methane.xyz --gen3d`
`xtb methane.xyz -ohess`
E_{0} (CH_{4}) = E_{0} (CH_{4}) - ZPE (CH_{4}) = -4.17521 Eh - -0.04482 Eh = -4.13039 Eh 
`obabel -:'[C]' -O carbon.xyz`
`xtb carbon.xyz --uhf 2`
E_{0} (C) = -1.79330 Eh
`obabel -:'[H]' -O hydrogen.xyz`
`xtb hydrogen.xyz --uhf 1`
4 * E_{0} (H) = 4 * -0.39348 Eh = -1.57392 Eh
\Delta E_{0} = -4.13039 Eh - -1.79330 Eh - -1.57392 Eh = -0.76317 Eh
DFT (B3LYP/6-31G*) expected \Delta E_{0} (same input files) = -0.62893 Eh

`obabel -:'CC' -O ethane.xyz --gen3d`
`xtb ethane.xyz -ohess`
E_{0} (C_{2}H_{6}) = E_{0} (C_{2}H_{6}) - ZPE (C_{2}H_{6}) = -6.27126 Eh - -0.04854 Eh =  -6.22271 Eh 
`obabel -:'[C]' -O carbon.xyz`
`xtb carbon.xyz --uhf 2`
2 * E_{0} (C) = 2 * -1.79330 Eh = -3.58660 Eh
`obabel -:'[H]' -O hydrogen.xyz`
`xtb hydrogen.xyz --uhf 1`
6 * E_{0} (H) = 6 * -0.39348 Eh = -2.36088 Eh
\Delta E_{0} = -4.13039 Eh - -1.79330 Eh - -1.57392 Eh = -0.27523 Eh
DFT (B3LYP/6-31G*) expected \Delta E_{0} (same input files) = -1.07149 Eh

Any help in finding my mistake or improving these calculations is great as even +/- 10 kCal/mol is very useful for my application. Thanks!

**Reply by haneug** [reply]
I did not check your calculations in detail but problems with the monoatomic species are not surprising. One can try to fix this by fitting atomic equivalent energies (https://doi.org/10.1021/acs.jcim.1c00312) or use low-level DFT singlepoints for the electronic energy (https://doi.org/10.1002/ange.202205735). A new xtb method that is currently in development should have no problems with this but it is not available to the public yet.

**Reply by njh80** [reply]
Thanks Hagen, I'll look at those methods and eyes peeled for the new xTB in the works!
